pkg/types/ccipocr3/interfaces.go

```diff
@@ -21,7 +21,8 @@ type MessageHasher interface {
 }
 
 type ExtraDataCodec interface {
-	DecodeExtraData(ExtraArgs Bytes, sourceChainSelector ChainSelector) (map[string]any, error)
+	DecodeExtraArgs(extraArgs Bytes, sourceChainSelector ChainSelector) (map[string]any, error)
+	DecodeDestExecData(destExecData Bytes, sourceChainSelector ChainSelector) (Bytes, error)
```

Comment:
To what format should this function decode to?
The input is []byte, and output is []byte, so its confusing.

Also, since this is []byte, it means theoretically a family can encode even a struct here, even though today EVM and SVM just use uint32. So I think, to handle any generic struct, we likely need the format similar to the ExtraArgsDecoded.

All the fields here in the generic Any2Any Message ideally should be in a format that is chain agnostic. So we can't have these in either source/destination specific coding.

Lets confirm with CCIP team if they agree here, or have other suggestions.

Reply:
Could you ask the onchain team? They confirmed the top level ExtraArgs is an object. Is this an object also?

At a minimum, I expect that this type should be `any` or `map[string]any`.